storage/mysql/storage.sql

```diff
@@ -22,8 +22,8 @@ CREATE TABLE IF NOT EXISTS Trees(
   Description           VARCHAR(200),
   CreateTimeMillis      BIGINT NOT NULL,
   UpdateTimeMillis      BIGINT NOT NULL,
-  PrivateKey            BLOB NOT NULL,
-  PublicKey             BLOB NOT NULL,
+  PrivateKey            MEDIUMBLOB NOT NULL,
```

Comment:
I'm not certain that 64KB is enough. It might be but the cost of avoiding that limit is low.